CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing
+
+We absolutely welcome any code contributions and we hope that this
+guide will facilitate an understanding of the PyAEDT antenna toolkit code
+repository. It is important to note that while the PyAEDT antenna toolkit
+software package is maintained by ANSYS and any submissions will be reviewed
+thoroughly before merging, we still seek to foster a community that can
+support user questions and develop new features to make this software
+a useful tool for all users. As such, we welcome and encourage any
+questions or submissions to this repository.
+
+For contributing to this project, please refer to the [PyAnsys Developer's Guide].
+
+[PyAnsys Developer's Guide]: https://dev.docs.pyansys.com/index.html
+
+## Directing Issues and Features Requests
+
+For reporting bugs and proposing new features, please use the Issues tab instead of the Discussions tab. This will help us track and prioritize work in a more organized manner.
```

Comment:
```suggestion
## Issues and features requests

To report bugs and propose new features, use the **Issues** page instead of the **Discussions** page. This helps us track and prioritize work in a more organized manner.
```